omero/developers/Tables.rst

```diff
@@ -241,8 +241,10 @@ Main methods
 
         :param long[] colNumbers: A list of column indices to be retrieved.
             The results will be returned in the same order as these indices.
+            If empty or None, all columns will be returned.
```

Comment:
Maybe go with `null` rather than `None` as this is not strictly Python based documentation?